### PR TITLE
Aligned pihole logo to center, Fixed responsive issue

### PR DESCRIPTION
**Pull request by Cafetier**
**By submitting this pull request, I confirm the following:** 
*please fill any appropriate checkboxes, e.g: [X]*

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [X] I have tested my proposed changes, and have included unit tests where possible.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

Please make sure you [Sign Off](https://github.com/pi-hole/pi-hole/wiki/How-to-signoff-your-commits.) all commits. Pi-hole enforces the [DCO](https://github.com/pi-hole/pi-hole/wiki/Contributing-to-the-project).

---
**What does this PR aim to accomplish?:**
*A detailed description, screenshots (if necessary), as well as links to any relevant GitHub issues*

**So basically**, the only thing it does is :
- vertically align the main logo on the page 
- make responsive the index (better than it was because personally on mobile, it's a pain in the a** to click on the link)

**How does this PR accomplish the above?:**
*A detailed description (such as a changelog) and screenshots (if necessary) of the implemented fix*

Well, added a couple of id and styles and poof, done (it only change html and css)

**What documentation changes (if any) are needed to support this PR?:**
*A detailed list of any necessary changes*

There is none
---
* You must follow the template instructions. Failure to do so will result in your pull request being closed.
* Please respect that Pi-hole is developed by volunteers, who can only reply in their spare time.

### CODE REVIEW

**Comment by dschaper** [comment]
Hello, can you please address the two failing test issues? Click on the Details to see the requirements.

And note that the blockpage will be going away as it's outlived it's usefulness.

**Comment by Cafetier** [comment]
@dschaper I have fixed both issues but for unknown reasons the test for support don't seam to pass :\ Do you have a fix ? Or whatever to make it work (before pushing my latest changes, it passed the supported distro test just fine)

**Comment by DL6ER** [comment]
> Step 1/9 : FROM buildpack-deps:xenial-scm
> unauthorized: authentication required
> ERROR: InvocationError for command '/usr/bin/docker build -f _ubuntu_16.Dockerfile -t pytest_pihole:test_container ../' (exited with code 1)
___________________________________ summary ____________________________________
ERROR:   py37: commands failed

This does not look like something you broke given that you didn't touch the test files.

**Comment by Cafetier** [comment]
@DL6ER Is there a way to restart those checks ? I guess by recommiting

**Comment by PromoFaux** [comment]
One of us just needed to press the button :)

**Review by dschaper — APPROVED** [review]
